LibreNMS/Snmptrap/Dispatcher.php

```diff
@@ -51,8 +59,9 @@ public static function handle(Trap $trap)
         // log an event if appropriate
         $fallback = $handler instanceof Fallback;
         $logging = Config::get('snmptraps.eventlog', 'unhandled');
+        $detailed = Config::get('snmptraps.eventlog_detailed', false);
         if ($logging == 'all' || ($fallback && $logging == 'unhandled')) {
-            Log::event("SNMP trap received: " . $trap->getTrapOid(), $trap->getDevice(), 'trap');
+            Log::event("SNMP trap received: " . $trap->toString($detailed), $trap->getDevice(), 'trap');
```

Comment:
Since the type is already "trap", can't we remove the "SNMP trap received"?

Reply:
Makes sense. I kept it because it was there but you are right.